skyportal/handlers/api/comment.py

```diff
@@ -214,4 +214,5 @@ def get(self, comment_id):
         self.set_header(
             "Content-Disposition", "attachment; "
             f"filename={comment.attachment_name}")
+        self.set_header("Content-type", "application/octet-stream")
```

Comment:
The omission of this line is what was causing the download comment attachment test to fail